src/commands/index.ts

```diff
@@ -36,6 +36,16 @@ export const commands: Command[] = [
       'The subscriber sound effect commands are now shared in one collective subscriber soundboard.'
     );
   }),
+  new SimpleCommand('!PBs', () => {
+    sayInChat(
+      'Wanna check out my PBs then go to https://www.speedrun.com/user/Dece'
```

Comment:
Can you format this the same as my !Github command? So just like
`'Wanna check out my PBs? https://www.speedrun.com/user/Dece'`